url-shortner-view/src/FirstView/FirstView.js

```diff
@@ -1,48 +1,52 @@
 import React, { useState } from 'react';
 import classes from './FirstView.module.css';
 import PostUrl from '../Service';
-import SecondView from '../SecondView/SecondView';
+import { useHistory } from 'react-router-dom';
 
-const FirstView = () => {
-	const [url, setUrl] = useState('');
+const FirstView = (props) => {
+  const history = useHistory();
+  const sendUrlHandler = (url) => {
+    if (url !== '') {
+      PostUrl(url)
+        .then((response) => {
+          props.setShortUrl(response.data.short_url);
+          console.log(props.shortUrl);
+        })
+        .then(() => history.push('/shortener'));
```

Comment:
since the first promise (then) isn't returning anything you can put this in the first then and no need for a second one